shell/platform/darwin/ios/ios_context_metal_impeller.mm

```diff
@@ -54,9 +56,9 @@
     int64_t texture_id,
     fml::scoped_nsobject<NSObject<FlutterTexture>> texture) {
   return std::make_unique<IOSExternalTextureMetal>(
-      fml::scoped_nsobject<FlutterDarwinExternalTextureMetal>{
-          [[darwin_context_metal_impeller_ createExternalTextureWithIdentifier:texture_id
-                                                                       texture:texture] retain]});
+      fml::scoped_nsobject<FlutterDarwinExternalTextureMetal>{[darwin_context_metal_impeller_
```

Comment:
Do we still need the scoped_nsobject here?

Reply:
Ideally the answer should be no, but since scoped_nsobject is ARC-friendly, we should be safe to do those in a followup patch if preferred.

Reply:
Should be fine to keep `scoped_nsobject`s since we have the multi-mode compliant version.